### ISSUE TITLE
[bug] rare bug where renaming a (parameter?) will cause ArrayIndexOutOfBoundsException

### ISSUE THREAD

**Issue by Decencies** [QuiltMC/enigma#89]
Occured when renaming parameter `x` (was previously var1, renamed to x) (also occurs when naming x as x)

![image](https://user-images.githubusercontent.com/66835910/230770300-c3880a04-2f5b-46d9-84e9-8fe0089efca9.png)

When the exception is ignored, and the (deobf) package tree is focused, the containing package for the class in which the exception occured (in this case: package `container` from `com/moonsworth/lunar/bridge/client/gui/`**container**`/GuiContainerBridge`) will no longer be represented as a package, but as a class.

![image](https://user-images.githubusercontent.com/66835910/230770434-7e04c1b7-686d-4aad-9fa3-05aeefef6010.png)

When the parent package (in this case: `gui`) is re-opened, it persists, until the class is re-named.

![java_pGnXiq2z3p](https://user-images.githubusercontent.com/66835910/230770521-2ec6f4b8-5178-4d7c-ac98-e5a6098de1a1.gif)

Trace:
```
java.lang.ArrayIndexOutOfBoundsException: 0 >= 0
	at java.base/java.util.Vector.elementAt(Vector.java:466)
	at java.desktop/javax.swing.tree.DefaultMutableTreeNode.getChildAt(DefaultMutableTreeNode.java:258)
	at java.desktop/javax.swing.tree.DefaultTreeModel.getChild(DefaultTreeModel.java:189)
	at java.desktop/javax.swing.tree.VariableHeightLayoutCache.createNodeAt(VariableHeightLayoutCache.java:764)
	at java.desktop/javax.swing.tree.VariableHeightLayoutCache.treeStructureChanged(VariableHeightLayoutCache.java:648)
	at java.desktop/javax.swing.plaf.basic.BasicTreeUI$Handler.treeStructureChanged(BasicTreeUI.java:4407)
	at java.desktop/javax.swing.tree.DefaultTreeModel.fireTreeStructureChanged(DefaultTreeModel.java:616)
	at java.desktop/javax.swing.tree.DefaultTreeModel.reload(DefaultTreeModel.java:320)
	at cuchaz.enigma.gui.ClassSelector.reload(ClassSelector.java:298)
	at cuchaz.enigma.gui.node.ClassSelectorClassNode$1.done(ClassSelectorClassNode.java:64)
	at java.desktop/javax.swing.SwingWorker$5.run(SwingWorker.java:750)
	at java.desktop/javax.swing.SwingWorker$DoSubmitAccumulativeRunnable.run(SwingWorker.java:848)
	at java.desktop/sun.swing.AccumulativeRunnable.run(AccumulativeRunnable.java:112)
	at java.desktop/javax.swing.SwingWorker$DoSubmitAccumulativeRunnable.actionPerformed(SwingWorker.java:858)
	at java.desktop/javax.swing.Timer.fireActionPerformed(Timer.java:311)
	at java.desktop/javax.swing.Timer$DoPostEvent.run(Timer.java:243)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:318)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:771)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:722)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:716)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:741)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
```
	
Version: latest

**Comment by tas-pro** [comment]
Same thing happened to me after I updated from b9edb8d6883daeae3c95681f61f9ba043d971673 to the latest commit.

**Comment by ix0rai** [comment]
to clarify, this wasn't happening before the merge of https://github.com/QuiltMC/enigma/pull/84?

**Comment by ix0rai** [comment]
@Decencies also, could you provide the jar and mappings that are causing the issue?

**Comment by Decencies** [comment]
> @Decencies also, could you provide the jar and mappings that are causing the issue?

here https://github.com/Lunar-Mapping-Project/mappings/releases/tag/93d4dd4%2Fmaster haha

**Comment by Decencies** [comment]
Whoops

**Comment by ix0rai** [comment]
can reproduce, thank you!

**Comment by ix0rai** [comment]
fixed in `1.7.6`.
